docs/ci-cd-environment/ubuntu-20.04-image.md

```diff
@@ -0,0 +1,218 @@
+---
+description: The ubuntu2004 is a customized image based on Ubuntu 20.04 LTS optimized for CI/CD. It comes with a set of preinstalled languages, databases, and utility tools.
+---
+
+# Ubuntu 20.04 Image
+
+
+The `ubuntu2004` is a customized image based on [Ubuntu 20.04 LTS](https://wiki.ubuntu.com/FocalFossa/ReleaseNotes)
+optimized for CI/CD. It comes with a set of preinstalled languages, databases,
+and utility tools commonly used for CI/CD workflows. The image can be paired
+with any [Linux machine type][machine-types] when defining the [agent][agent]
+of your pipeline or block.
+
+The `ubuntu2004` is a virtual machine (VM) image. The user in the environment,
+named `semaphore`, has full `sudo` access. The image will be updated bi-weekly, on first and third Monday of every month.
+Updates can be followed on [Semaphore Changelog](https://docs.semaphoreci.com/reference/semaphore-changelog/).
+
+The `ubuntu2004` VM uses an *APT mirror* that is in the same data center as
+Semaphore's build cluster, which means that caching packages will have little
+effect.
+
+## Using the ubuntu2004 OS image in your agent configuration
+
+To use the `ubuntu2004` OS image, define it as the `os_image` of your agent's
+machine.
+
+``` yaml
+version: 1.0
+name: Ubuntu18 Based Pipeline
+
+agent:
+  machine:
+    type: e1-standard-2
+    os_image: ubuntu2004
+
+blocks:
+  - name: "Unit tests"
+    task:
+      jobs:
+        - name: Tests
+          commands:
+            - make test
+```
+
+The `ubuntu2004` OS image can be used in combination with all Linux machine
+types: `e1-standard-2`, `e1-standard-4`, `e1-standard-8`.
```

Comment:
@bogyo210 only **e1-standard-2** will be available at the start. Should be added as a note. (check [this line](https://github.com/semaphoreci/docs/blame/master/docs/ci-cd-environment/custom-ci-cd-environment-with-docker.md#L49) to see how to add notes)